### DISCUSSION TITLE
Can I connect to a broker using Client Certificates ?

### DISCUSSION THREAD

**Discussion by rido-min** [EdJoPaTo/mqttui#119] [Q&A]
I'd like to use mqttui with a broker that is only configured to accept client certificates for authentication, these can be provided as PEM/KEY files, or PFX.

**Comment by EdJoPaTo** [comment]
Yes and no. This was added in #102 so you can when you use the version from the main branch. But this is not released into a new version yet. (I wanted to work on some things before a new release but that hasn't happened yet…)

You can download the prebuilt binaries from [the GitHub actions](https://github.com/EdJoPaTo/mqttui/actions/workflows/rust.yml) or clone the repo and build it yourself (`cargo install --path .`)

Would be nice to know to if it works as intended. 😇
